imap_processing/ultra/l1c/pset.py

```diff
@@ -0,0 +1,131 @@
+"""Calculates Pointing Set Grids."""
+
+import numpy as np
+
+
+def calculate_pset_exposure(l1b_de_dict, l1b_extendedspin_dict, l1b_cullingmask_dict):
+    """
+    Create dictionary with defined datatype for Pointing Set Grid Data.
+
+    Parameters
+    ----------
+    l1b_de_dict: : dict
```

Comment:
Extra colons on all of these. Also suggest dataset
```suggestion
    l1b_de_dict : dict
```